docs/core/compatibility/interop/5.0/casting-rcw-to-inspectable-interface-throws-exception.md

```diff
@@ -13,7 +13,7 @@ Casting a runtime callable wrapper (RCW) to an interface marked as <xref:System.
 
 ## Change description
 
-In .NET versions prior to .NET 5 preview 6, casting an RCW to an interface marked as <xref:System.Runtime.InteropServices.ComInterfaceType.InterfaceIsIInspectable> works as expected. In .NET 5 previews 6-8 and RC1, you can successfully cast an RCW to an <xref:System.Runtime.InteropServices.ComInterfaceType.InterfaceIsIInspectable> interface. However, you might get access violations when you execute methods on the interface, because the underlying support in the runtime [was removed in .NET 5 preview 6](built-in-support-for-winrt-removed.md).
+In .NET versions prior to .NET 5 Preview 6, casting an RCW to an interface marked as <xref:System.Runtime.InteropServices.ComInterfaceType.InterfaceIsIInspectable> works as expected. In .NET 5 Previews 6-8 and RC1, you can successfully cast an RCW to an <xref:System.Runtime.InteropServices.ComInterfaceType.InterfaceIsIInspectable> interface. However, you might get access violations when you execute methods on the interface, because the underlying support in the runtime [was removed in .NET 5 Preview 6](built-in-support-for-winrt-removed.md).
```

Comment:
Why keep preview-number-specific text here?

Reply:
I noticed a number of .NET 5 breaking changes that are preview-to-preview only. I need to decide if I'm going to remove them all, but if I do, it will be in a separate PR.

But to answer your question, without the preview numbers here, I don't think this would make any sense.

Reply:
Would this rewrite make sense and be accurate?
```suggestion
In .NET versions prior to .NET 5, casting an RCW to an interface marked as <xref:System.Runtime.InteropServices.ComInterfaceType.InterfaceIsIInspectable> works as expected. In .NET 5, you can successfully cast an RCW to an <xref:System.Runtime.InteropServices.ComInterfaceType.InterfaceIsIInspectable> interface. However, you might get access violations when you execute methods on the interface, because the underlying support in the runtime [was removed](built-in-support-for-winrt-removed.md).
```

Reply:
You're right, this breaking change is not preview to preview. However your suggested text doesn't take into account the final behavior that was introduced in RC 2, so I've committed some new text.